ci/test-alive-on-cluster.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -e
+
+cd "$( dirname "${BASH_SOURCE[0]}" )"
+
+set +e
```

Comment:
Why all the `set -e`, `set +e` dance? Leave echo mode on (`set -e`) and drop the echo's, it won't be much more spammy, and the code will be more readable.